### PR TITLE
435-add-tag-details-to-page-metadata

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://435-add-tag-details-to-page-metadata--danaher-ls-aem--hlxsites.hlx.page/

### CODE REVIEW

**Review by davenichols-DHLS — APPROVED** [review]
Approved.

File: tools/importer/transformers/metadata.js [hunk 1]
```diff
@@ -77,6 +77,11 @@ const createMetadata = (main, document, html, params, url) => {
     meta.keywords = keywords.content;
   }
 
+  const tags = document.querySelector('[name="tags"]');
+  if (tags) {
+    meta.Tags = tags.content;
```
Comment (duynguyen) [inline tools/importer/transformers/metadata.js:82]: if tags meta has no content, do we still want to add it?

Comment (mhaack) [reply tools/importer/transformers/metadata.js:82]: No we only should add meta data if they have some value.

Comment (sharanagoudapatil) [reply tools/importer/transformers/metadata.js:82]: @mhaack & @duynguyen - if there is no content then will not add placeholder with empty values. It is handled now. Please verify. Thanks.

**Comment by mhaack** [comment]
@sharanagoudapatil the PR looks ok to me now. However I think we should directly add a second meta data with the optc/brand which we extract from the tags. We need to later for the product home page and/or the opco pages.

@davenichols-DHLS WDYT? What are we going to use the tags for? I understood a meta field "brand" or "opco" would be needed. We can also go with generic tags and later filter that in the UI.

**Review by duynguyen — APPROVED** [review]

**Comment by sharanagoudapatil** [comment]
Hi @mhaack - Yes you are correct, getting opco/brand from the tags itself but this field was coming from components like articleInfo, articleCard. Sure, will discuss with @davenichols-DHLS once for further changes. Thanks.

**Comment by mhaack** [comment]
For the tags we get `<meta name="tags" content="danaher:product/categories/microscopes,danaher:opco/leica-microsystems,danaher:workflows/plasmid-dna/cell-recovery"/>` and we can extract the opco from it as it always starts with "danaher:opco" and store it into its own field.

**Pull request merged**